ovos_m2v_pipeline/__init__.py

```
@@ -14,236 +11,439 @@
-            A list of Padatious intent names not present in the ignore list.
-
-        Raises:
-            RuntimeError: If no response is received from the bus.
-        """
         msg = Message("intent.service.padatious.manifest.get")
         res = self.bus.wait_for_response(msg, "intent.service.padatious.manifest", timeout=timeout)
         if not res:
             raise RuntimeError("Failed to retrieve intent names")
         return [i for i in res.data["intents"] if i not in self.ignore_labels]
 
     def handle_sync_intents(self, message: Message) -> None:
```

Ruff (0.15.2)

[warning] Unused method argument: `message`

(ARG002)